samples/visualization_constraints.py

```diff
@@ -107,6 +107,14 @@
                                       direction=1, radius=15, tube_radius=6),
         particle_type=0, penetrable=True)
 
+elif args.shape == "Quarterpipe":
+    system.constraints.add(
+        shape=espressomd.shapes.Quarterpipe(center=[25, 15, 15], axis=[1, 0, 0],
+                                            orientation=[0, 1, 1], radius=25,
+                                            height=30),
+        particle_type=0, penetrable=True)
```

Comment:
* I'm not sure it is a good idea to have two sharp edges in a shape interacting with a LJ liquid; wouldn't it be safer to have e.g. 4 quaterpipes to form a pipe, or a quaterpipe with its X,Y,Z edge lengths equal to the box lengths? 
* the rasterization doesn't draw dots on the shape top and bottom surfaces, giving the impression the shape is hollow.

Reply:
the shape becomes unsafe when side walls touch the periodic boundaries. Then Particles can enter the interaction region from the other side and pick up a lot of potential energy that propels them through the box. I will change the ``height`` though to hide the top/bottom